PPPort_pm.PL

```diff
@@ -711,7 +711,7 @@ package Devel::PPPort;
 use strict;
 use vars qw($VERSION $data);
 
-$VERSION = '3.55_01';
+$VERSION = '3.56';
```

Comment:
this is wrong we need 3.55
note I've adjusted Makefile.PL to work with devel version view 91191d306e83fed5b8f4d58e42a9ffacd4e1fce7